libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.m

```diff
@@ -1861,14 +1861,12 @@ - (void)presentLoginView:(SFSDKAuthViewHolder *)viewHandler {
             }];
         }
         else {
-#if __IPHONE_OS_VERSION_MIN_ALLOWED >= 130000
```

Comment:
@trooper2013, thanks for making this change. I am not sure why if using __IPHONE_OS_VERSION_MIN_ALLOWED >= 130000 to do conditional logic based on deployment target is the right thing to do in general, should we do an inventory to see if we should remove from other places?

Reply:
@qingqingliu These are the only special cases.